transcom/mymove
# [MB-1752] Import May 2020 discount rates

## Description

This PR contains a migration to [import the discount rates](https://github.com/transcom/mymove/blob/master/docs/data/tspp-data-creation.md) for the second performance period of 2020 (May 15-Jul 31). A local migration with scrubbed data is included in the PR, and a secure migration for deployed environments has been uploaded to S3.

## Reviewer Notes

I plan to deploy this to experimental after getting an initial thumbs up on the PR to make sure it works OK there before it lands in staging/prod.

## Setup

To test locally (your `dev_db` database):
- `make db_dev_reset db_dev_migrate`

To test deployed migrations (your `deployed_migrations` database):
- `make run_prod_migrations` (or `run_staging_migrations` or `run_experimental_migrations`)

You should find 847 `transportation_service_provider_performances` records with a `performance_period_start` of `2020-05-15` in the databases above.

Please also spot check the discount rates in those records vs. those found in the data files linked below.

You can also try setting up a PPM move with a move date that falls within this performance period and make sure that it seems to estimate it OK.

## Code Review Verification Steps

* Any new migrations/schema changes:

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-1752) for this change
* [Directory with discount rates](https://drive.google.com/drive/u/0/folders/1T9qmsthhg9fAXkIxC55CAQ5SFHSnqLA5)
* [Directory with BVS/SCAC info](https://drive.google.com/drive/u/0/folders/1Es3Ld-ouqH9JbM42plr9fmNwxcUlbOqP)
* [Instructions for importing discount rates](https://github.com/transcom/mymove/blob/master/docs/data/tspp-data-creation.md)
